src/test/java/bio/terra/common/auth/AuthService.java

```diff
@@ -27,66 +22,57 @@
     private List<String> directAccessScopes = Arrays.asList(
         "https://www.googleapis.com/auth/bigquery",
         "https://www.googleapis.com/auth/devstorage.full_control");
-    private NetHttpTransport httpTransport;
-    private JacksonFactory jsonFactory = JacksonFactory.getDefaultInstance();
-    private File pemfile;
     private String saEmail;
-    private Map<String, String> userTokens = new HashMap<>();
-    private Map<String, String> directAccessTokens = new HashMap<>();
     private TestConfiguration testConfig;
 
+    private Map<String, GoogleCredentials> userCreds = new HashMap<>();
+    private Map<String, GoogleCredentials> directAccessCreds = new HashMap<>();
+
     @Autowired
     public AuthService(TestConfiguration testConfig) throws Exception {
         this.testConfig = testConfig;
-        Optional<String> pemfilename = Optional.ofNullable(testConfig.getJadePemFileName());
-        pemfilename.ifPresent(s -> pemfile = new File(s));
         saEmail = testConfig.getJadeEmail();
-        httpTransport = GoogleNetHttpTransport.newTrustedTransport();
     }
 
     public String getAuthToken(String userEmail) {
-        if (!userTokens.containsKey(userEmail)) {
-            userTokens.put(userEmail, makeToken(userEmail));
-        }
-        return userTokens.get(userEmail);
+        return getCredentials(userEmail).getAccessToken().getTokenValue();
     }
 
     public String getDirectAccessAuthToken(String userEmail) {
-        if (!directAccessTokens.containsKey(userEmail)) {
-            directAccessTokens.put(userEmail, makeDirectAccessToken(userEmail));
+        if (!directAccessCreds.containsKey(userEmail)) {
+            directAccessCreds.put(userEmail, makeDirectAccessCredentials(userEmail));
         }
-        return directAccessTokens.get(userEmail);
+        return directAccessCreds.get(userEmail).getAccessToken().getTokenValue();
     }
 
-    private GoogleCredential buildCredential(String email, List<String> scopes)
```

Comment:
I could probably revert all the changes in this file and do the `GoogleCredentials` setup manually in the test where I need it. I'm not sure if it's the cause of the failures in the connected tests, though